src/katgpucbf/xbgpu/bsend.py

```diff
@@ -344,9 +344,9 @@ class BSend:
     ----------
     outputs
         Sequence of :class:`.output.BOutput`.
-    frames_per_chunk
-        Number of :class:`Frame`\ s in each transmitted :class:`Chunk`.
-    n_tx_items
+    batches_per_chunk
+        Number of :class:`Batch`\ es in each transmitted :class:`Chunk`.
+    n_out_items
```

Comment:
I think this should maybe get called `n_chunks`, since this is not the number of BOutQueueItem instances.